test/trento/settings_test.exs

```diff
@@ -1,5 +1,6 @@
 defmodule Trento.SettingsTest do
   use ExUnit.Case
+  alias Trento.Settings.ActivityLogSettings
```

Comment:
nitpick: I think we can group this alias below, as we already have the `Trento.Settings` aliased and grouped

Reply:
Yup, thanks https://github.com/trento-project/web/pull/3223/commits/779e30d98bbf46f5cd516b7be66038b28195e740